pkg/client/fake/client.go

```diff
@@ -417,8 +428,7 @@ func (c *fakeClient) Scheme() *runtime.Scheme {
 }
 
 func (c *fakeClient) RESTMapper() meta.RESTMapper {
-	// TODO: Implement a fake RESTMapper.
-	return nil
+	return c.restMapper
```

Comment:
We could default to `meta.NewDefaultRESTMapper([]schema.GroupVersion{})`. Which is essentially an empty RESTMapper.

I'm not aware of any fake RESTMapper implementations but I think the meta. DefaultRESTMapper comes very close as it's essentially just a bunch of (local) maps.

xref: https://github.com/kubernetes-csi/driver-registrar/blob/795af1899f3c94dd0c6dda2a25ed301123479bb9/vendor/k8s.io/apimachinery/pkg/api/meta/restmapper.go#L57-L65

The only problem is that the interface only has read-only funcs so it's not possible to fill the rest mapper when we default it (https://github.com/kubernetes-csi/driver-registrar/blob/795af1899f3c94dd0c6dda2a25ed301123479bb9/vendor/k8s.io/apimachinery/pkg/api/meta/interfaces.go#L113-L134). But I guess it would be better than just nil.